transform.js

```diff
@@ -133,7 +133,10 @@ const toSchemaObject = definition => {
  * @return     {object}  A plain JavaScript object which conforms to JSON Schema
  */
 const transform = document => {
-  const definitions = document.definitions.map(toSchemaObject);
+  // ignore directives
+  const definitions = document.definitions.
+    filter(d => d.kind !== 'DirectiveDefinition').
+    map(toSchemaObject);
```

Comment:
This is just a small style nitpick but any chance you could put the `.` before `filter` and `map`? 
Just for consistency – I should really put a linter in the build chain.

Reply:
@jakubfiala done ! 👍